### ISSUE TITLE
Module level comments not showing up?

### ISSUE THREAD

**Issue by aayla-secura** [krisztianb/typedoc-plugin-merge-modules#24]
Hi there, I'm experiencing a similar issue to the already closed #9 . I don't know if I'm doing something wrong though, or if it's a new bug.

I'm using `mergeModulesMergeMode: module`. I have several files with the same `@module` name and I want the module page to have a comment. I tried the following in one of the files (the others just have the `@module SomeModule` line).

file 1:
```
/**
 * @module SomeModule
 * @packageDocumentation
 * @mergeTarget
 * 
 * Some comment
 */
```

I get the following error:

```
Encountered an unknown block tag @mergeTarget
```

If I do this:
```
/**
 * @module SomeModule
 */

/**
 * @module SomeModule
 * @packageDocumentation
 * @mergeTarget
 * 
 * Some comment
 */
```

Then the error goes away, but it seems like the second block is ignored and the comment doesn't show up anyway.

Using version 6.1.0 with Typedoc 0.26.11

**Comment by krisztianb** [comment]
Hi. Do you have a repo where I can reproduce your issue. If not please post your TypeDoc config and call.

> I get the following error:
> Encountered an unknown block tag @mergeTarget

This is because this plugin uses a made up jsdoc block tag `@mergeTarget`. That should only be a "warning" a not cause any problems.

**Comment by aayla-secura** [comment]
Oh, I see! I thought that the warning meant it wasn't working at all. I'm guessing these can't warnings be suppressed?

I experimented a bit and turns out I also need `@preferred` tag to make the comments show up. With `@preferred` and `@mergeTarget` it works :)

**Comment by krisztianb** [comment]
Can you give me a link to this `@preferred` tag? I've never heard of it and can't find anything about it.

Btw. you might not need this plugin at all because TypeDoc 0.27 introduced the tag [`@mergeModuleWidth`](https://typedoc.org/documents/Tags._mergeModuleWith.html).

**Comment by aayla-secura** [comment]
Oh, that's interesting! I read about the `@preferred` tag in issue #9 that I linked to and I tried it and it worked. Without it, package comments are not showing up. Have a look at https://github.com/aayla-secura/typedoc-repros/tree/gh-plugin-merge-modules-24

"AnotherMergedModule" (multiple files) and "YetAnotherModule" (single file) are missing `@preferred` and the doc generated doesn't have the comment at the top.

____

I did come across `@mergeModuleWith` but there was a warning on the doc page that links will not be properly resolved, so I didn't try it.
